src/converse-profile.js

```diff
@@ -257,9 +257,23 @@ converse.plugins.add('converse-profile', {
             },
 
             afterRender () {
+                this.updateStatusDisplayForSelfContact();
                 this.renderAvatar();
             },
 
+            updateStatusDisplayForSelfContact () {
+                const self = this;
```

Comment:
Please don't do this. You can use an arrow function below to ensure that `this` is bound to the right context.